test/GraphTest.cpp

```diff
@@ -105,6 +105,39 @@ TEST(GraphTest, GetNodeSet_2) {
               }) != nodeSet.end());
 }
 
+TEST(GraphTest, FindEdge_Test) {
```

Comment:
Please add spacing in logical places between lines in this test. It's hard to read

Reply:
Done. Please review.